### PR TITLE
Permit to upload a SVG file as custom icon

### PR DESCRIPTION
**Pull request by Progi1984**
Fixes PrestaShop/PrestaShop#15939

Need PrestaShop/PrestaShop#16316

### CODE REVIEW

File: controllers/admin/AdminBlockListingController.php [hunk 1]
```diff
@@ -140,7 +140,12 @@ public function displayAjaxSaveBlockContent()
             $filename = $customImage['name'];
 
             // validateUpload return false if no error (false -> OK)
-            $validUpload = ImageManager::validateUpload($customImage);
+            $validUpload = ImageManager::validateUpload(
+                $customImage,
+                0,
+                ['gif', 'jpg', 'jpeg', 'jpe', 'png', 'svg'],
+                ['image/gif', 'image/jpg', 'image/jpeg', 'image/pjpeg', 'image/png', 'image/x-png', 'image/svg']
```
Comment (PierreRambaud) [inline controllers/admin/AdminBlockListingController.php]: if it needs a change on the develop branch, you must update compatibility to 1.7.7.0 :)

Comment (Progi1984) [reply controllers/admin/AdminBlockListingController.php]: Fixed :heavy_check_mark:

**Comment by PierreRambaud** [comment]
Thanks @Progi1984

**Comment by PrestaShark** [comment]
Sorry but classes/ReassuranceActivity.php **getMimeType** should be replaced by **getMimeTypeByExtension**. Thanks

**Pull request merged**